app/src/main/java/com/example/appempty/SecondFragment.kt

```diff
@@ -59,15 +56,19 @@ class SecondFragment : Fragment() {
             }
 
             override fun onResponse(call: Call<RandomUserResponse>, response: Response<RandomUserResponse>) {
-                val results = response.body() ?: throw Throwable("error1")
-
-                    fullname.setText(results.User().fullName())
-                    username.setText(results.User().Login().username)
-                    state.setText(results.User().Location().state)
-                    email.setText(results.User().email)
-                    phone.setText(results.User().phone)
-                    GlideApp.with(context)
-                    .load(results.User().picture)
+                    response.isSuccessful ?: throw Throwable("error1")
+
+                    fullname.setText(response.body()!!.results[0].name.first)
```

Comment:
response.body()!!.results[0] this could be saved in a val, so you do not carry with the !! check many times... also you need to set a default value for error case.
response.body()?.results[0] ?: "empty"